client/client/main.go

```diff
@@ -46,27 +47,40 @@ func periodicAdd(cli *client.Client) {
 		currTime := time.Now()
 
 		var lastIndexed time.Time
-		lastIndexedJSON, err := ioutil.ReadFile(filepath.Join(*clientDirectory, ".timestamp"))
+		f, err := os.OpenFile(filepath.Join(*clientDirectory, ".search_kbfs_timestamp"), os.O_RDWR|os.O_CREATE|os.O_EXCL, 0666)
```

Comment:
I don't think O_EXCL works for the last indexer, because now the file will only ever get written once!  The previous way was fine.